### PR TITLE
Refactor Equipment and BrainRegion table assignments (monkey patching fix)

### PR DESCRIPTION
**Pull request by tdincer**

### CODE REVIEW

File: workflow_calcium_imaging/__init__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+try:
+    import matlab.engine
+except Exception:
+    pass
```
Comment (ttngu207) [inline workflow_calcium_imaging/__init__.py:4]: You probably don't need this

**Review by ttngu207 — APPROVED** [review]

File: workflow_calcium_imaging/__init__.py (hunk 1 above)
Comment (tdincer) [reply workflow_calcium_imaging/__init__.py:4]: Thats's in case of EXTRACT.

File: workflow_calcium_imaging/__init__.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+import os
+import datajoint as dj
+
+if "custom" not in dj.config:
+    dj.config["custom"] = {}
+
+# overwrite dj.config['custom'] values with environment variables if available
```
Comment (CBroz1) [inline workflow_calcium_imaging/__init__.py:12]: What's the motivation for these adds?

Comment (tdincer) [reply workflow_calcium_imaging/__init__.py:12]: All the changes are transferred from `wt-calcium-imaging` as it is the most cloud-tested calcium imaging workflow.

Comment (tdincer) [reply workflow_calcium_imaging/__init__.py:12]: The priority is given to environment variables. If the variables are not available, these adds set the defaults.

File: workflow_calcium_imaging/reference.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+@schema
+class Equipment(dj.Lookup):
+    definition = """
+    scanner: varchar(32)
+    """
+    contents = zip(["ScannerA"])
+
+
+@schema
```
Comment (CBroz1) [inline workflow_calcium_imaging/reference.py:16]: Should we pull from the new element-animal surgery schema instead? [BrainRegion table](https://github.com/datajoint/element-animal/blob/main/element_animal/surgery.py#L77-L90)

**Comment by tdincer** [comment]
All changes suggested here were applied in another PR.

**Pull request closed without merging**